data/web/inc/functions.inc.php

```diff
@@ -1183,4 +1183,19 @@ function get_logs($container, $lines = false) {
   }
   return false;
 }
-?>
+function getGUID() {
+  if (function_exists('com_create_guid')) {
+    return com_create_guid();
+  }
+
+  mt_srand((double)microtime()*10000);//optional for php 4.2.0 and up.

```

Comment:
What is that good for?

Reply:
Seeds the Mersenne Twister Random Number Generator

Reply:
but for just this project we can use just com_create_guid()